contrib/thermald/template.py

```diff
@@ -0,0 +1,44 @@
+pkgname = "thermald"
+pkgver = "2.5.4"
+pkgrel = 0
+archs = [
+    "x86_64",
+]
+build_style = "gnu_configure"
+make_cmd = "gmake"
+hostmakedepends = [
+    "autoconf",
+    "autoconf-archive",
+    "automake",
+    "gettext",
+    "gmake",
+    "gtk-doc-tools",
+    "libtool",
+    "pkgconf",
+]
+makedepends = [
+    "dbus-devel",
+    "dbus-glib-devel",
+    "glib-devel",
+    "libevdev-devel",
+    "xz-devel",
+    "libxml2-devel",
+    "upower-devel",
+]
+pkgdesc = "Thermal daemon for x86_64-based Intel CPUs"
+maintainer = "miko <mikoxyzzz@gmail.com>"
+license = "GPL-2.0-or-later"
+url = "https://github.com/intel/thermal_daemon"
+source = f"https://github.com/intel/thermal_daemon/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "2addedfc7974b30ed6a553a5690961eb48d6abacd4584e3b6d3fd292343d694e"
+hardening = ["vis"]
+
+
+# necessary since we don't want to run thermald's silly autogen
```

Comment:
the autogen looks fine though, you could do `configure_gen = ["NOCONFIGURE=1", "./autogen.sh"]` or whatnot

Reply:
nope, doesn't work; it'll try to run `NOCONFIGURE=1` as a command instead of interpreting it as an env variable :<

i did try to add `NOCONFIGURE=1` to env but that seemed to screw some things up (specifically `MAKE` wasn't set in do_build for whatever reason) :<

Reply:
> nope, doesn't work; it'll try to run NOCONFIGURE=1 as a command instead of interpreting it as an env variable :<  

ah right. you want `configure_env = {"NOCONFIGURE": "1"}`

Reply:
you are right it funnily breaks though

Reply:
!

i'll fix that tomorrow. thanks!

Reply:
yeah idk why it's so broken, but you can use just

```
build_style = "gnu_configure"
configure_args = ["--disable-dependency-tracking"]
configure_env = {"NO_CONFIGURE": "1"}
configure_gen = ["./autogen.sh"]
make_cmd = "gmake"
```

and that works i guess